recipe/meta.yaml

```diff
@@ -1,46 +1,43 @@
 {% set name = "pyload-ng" %}
-{% set version = "0.5.0b3.dev85" %}
+{% set version = "0.5.0b3.dev87" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 28ba5f879dc92aa7b49192dc4350b87d714561e3b9a4f8626bd7d1c56b1b2076
+  sha256: 2db8b16947ec59e53c595a59b122c998547f8552e10abd4346c21e15809cc450
 
 build:
   entry_points:
     - pyload = pyload.__main__:main
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  noarch: python
 
 requirements:
   host:
     - pip
-    - python
+    - python <3.12 #see PR and issues 58 & 57
     - setuptools >=38.3
     - wheel
   run:
-    - python
-    - cheroot >=8.4,<9.dev0
+    - python <3.12
+    - cheroot
     - flask
     - flask-babel >=1.0,<2.dev0
     - flask-caching >=1.9,<2.dev0
     - flask-compress >=1.8,<2.dev0
-    - flask-session >=0.4.1,<0.5.dev0  #Should be for 37 or less according to grayskull, but not according to the tests
```

Comment:
`- flask-session >=0.4.1,<0.5.dev0  #Should be for 37 or less according to grayskull, but not according to the tests`